app/components/login.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
```

Comment:
I'm working on UI

Reply:
oh,sorry you are right, but i couldnt test it with out login component

Reply:
ok. we can figure out how to integrate my code into this tomorrow. For now I'll leave the PR open.